<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config_editor</class>
 <widget class="QMainWindow" name="config_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout_3">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget_1">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Main Config</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="error">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Connection Info</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Server Port</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Server IP</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="serverIP"/>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="serverPort1"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Contestant Config </string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLineEdit" name="constConf"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Presenter Config</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLineEdit" name="presenterConf"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Admin Config</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="adminConf"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Server Config</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Server Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverPort2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Server Config File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverConf"/>
        </item>
        <item>
         <widget class="QLabel" name="label_sql">
          <property name="text">
           <string>SQL File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sql"/>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Round1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_q1">
              <property name="text">
               <string>Question File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="q_rnd1"/>
            </item>
            <item>
             <widget class="QLabel" name="label_a1">
              <property name="text">
               <string>Answer File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="a_rnd1"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Round2</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_q2">
              <property name="text">
               <string>Question File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="q_rnd2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_a2">
              <property name="text">
               <string>Answer File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="a_rnd2"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Round3</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_q3">
              <property name="text">
               <string>Question File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="q_rnd3"/>
            </item>
            <item>
             <widget class="QLabel" name="label_a3">
              <property name="text">
               <string>Answer File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="a_rnd3"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Round4</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_q4">
              <property name="text">
               <string>Question File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="q_rnd4"/>
            </item>
            <item>
             <widget class="QLabel" name="label_a4">
              <property name="text">
               <string>Answer File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="a_rnd4"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="load">
      <property name="text">
       <string>                        Load                     </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
